script/DeployServiceFeeCollector.s.sol

```diff
@@ -13,7 +13,10 @@ contract DeployScript is DeployScriptBase {
     {
         vm.startBroadcast(deployerPrivateKey);
 
-        constructorArgs = abi.encode(deployerAddress);
+        address owner = address(0x8932eb23BAD9bDdB5cF81426F78279A53c6c3b71);
+        // or for testing:
+        // address owner = deployerAddress
+        constructorArgs = abi.encode(owner);
```

Comment:
This is not so clean, perhaps you know a better way to specify that address in env? I wanted to avoid having to move access for every contract manually after deployment.

Reply:
How about you use the foundry cheatcode to derive the address from your private key? That way the script should work for both TesterWallet and DeployerWallet

https://book.getfoundry.sh/cheatcodes/addr

Reply:
Also I noticed that you want to merge this branch into master directly. 
Does our management of the various branches also include that anything that gets merged into master directly will also be merged into (staging and) dev?
It should imho....otherwise we will have diverging codebases...

Reply:
Accessing the deployer wallet is no problem, that is what was done before using the `deployerAddress` address. I want to use a third address which should later collect the fees.

Reply:
I created this PR agains master because I added the contracts to the production diamond. We have to manually merge master into the other branches afterwards. I chose this direction as everything from master should be good for the other environments, but we don't want to merge everything from dev/staging to master right away.